app/res/layout/home_sync_button.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<org.commcare.android.view.SquareButtonWithNotification
```

Comment:
This might be out of scope for day 1, but is it possible for us to create these buttons programatically? It seems like it might be easier to extend if we could initialize them with text+backgroundcolor+image, and it seems like they have roughly the same layouts? Would make it easier for us to do more with the homescreen a bit faster.